MANIFEST.in

```diff
@@ -1,19 +1,23 @@
 include LICENSE
 include README.md
+include CHANGELOG.md
```

Comment:
We can even combine this line with the one above with `*.md`.

Reply:
Yes, but I wanted to add both separately to know exactly what we need.
Is the RELEASE.md necessary?

Reply:
Ah right. Then I guess it's fine to leave it like it is now.